webdriver/tests/bidi/script/add_preload_script/contexts_tentative.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+async def test_add_preload_script_specify_context_multiple_times(
```

Comment:
I would rto ename the test so that it is clear what it actually tests:

```suggestion
async def test_identical_contexts(
```